.ci/setup_env_centos.sh

```diff
@@ -29,7 +29,8 @@ sudo -E dnf -y install epel-release
 # Enable priority to CentOS Base repo in order to
```

Comment:
I wonder if that block is really needed for CentOS Stream at all. I bet there are many tweaks carried on the `setup_env_centos.sh` not needed, so I suggest we start with a almost fresh script.

Reply:
TBH i thought about it but i was lazy and assumes (maybe wrongly) CentOS stream is closer to Centos than Fedora so it's needed 
But wouldn't it make sense to drop CentOS in favor of CentOS-Stream later and just use the same script (dropping what's not needed)?

Reply:
^ @wainersm